changelog.txt

```diff
@@ -1,3 +1,7 @@
+V.Next 2
```

Comment:
V.Next 2?

Reply:
Updated (I added this when vNext still contained things that were part of current release).